src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -763,6 +764,13 @@ const WorkflowView = _.flow(
                 href: 'https://support.terra.bio/hc/en-us/articles/360039681632',
                 ...Utils.newTabLinkProps
               }, ['Click here to learn more.'])
+            ]),
+            span({ style: { margin: '0 0.5rem 0 1rem' } }, [
+              h(LabeledCheckbox, {
+                checked: useReferenceDisks,
+                onChange: v => this.setState({ useReferenceDisks: v }),
+                style: { marginLeft: '1rem' }
+              }, [' Use reference disks'])
```

Comment:
Is that space before `Use` supposed to be there?

Reply:
Yes, the `LabeledCheckbox` component puts the checkbox inline with the text, so we typically put a space to create a small gap between them.